v22.2/use-cloud-storage-for-bulk-operations.md

```diff
@@ -27,7 +27,7 @@ URLs for the files you want to import must use the format shown below. For examp
 
 Location                                                    | Scheme      | Host                                             | Parameters                                                                 
 ------------------------------------------------------------+-------------+--------------------------------------------------+----------------------------------------------------------------------------
-Amazon                                                      | `s3`        | Bucket name                                      | `AUTH`: optional `implicit` or `specified` (default: `specified`); `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, [`AWS_SESSION_TOKEN`](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_use-resources.html). For more information, see [Authentication - Amazon S3](#authentication). <br><br>`S3_STORAGE_CLASS`: Specify the Amazon S3 storage class for created objects. **Default**: `STANDARD`. See [Amazon S3 storage classes](#amazon-s3-storage-classes) for the available classes.                              
+Amazon                                                      | `s3`        | Bucket name                                      | [`AUTH`](#authentication): `implicit` or `specified` (default: `specified`). When using `specified` pass user's `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`.<br><br>[`ASSUME_ROLE`](#assume-role-authentication) (optional): Pass the [ARN](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html) of the role to assume. Use in combination with `AUTH=implicit` or `specified`.<br><br>[`AWS_SESSION_TOKEN`](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_use-resources.html) (optional): For more information, see [Authentication - Amazon S3](#authentication). <br><br>`S3_STORAGE_CLASS` (optional): Specify the Amazon S3 storage class for created objects. **Default**: `STANDARD`. See [Amazon S3 storage classes](#amazon-s3-storage-classes) for the available classes.                              
```

Comment:
Nit: suggest that all leading new concepts in the `Parameters` column for the `Amazon` entry link to internal resources, with links to third-party supporting docs appearing only in their descriptions that follow. As it stands: 2 are internal, 1 is external, and 1 is unliked. The following I think is more intuitive (also trying out bullets):

```
- [`AUTH`](#authentication): ...
- [`ASSUME_ROLE`](#assume-role-authentication): ... Pass the [ARN](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html) ...
- [`AWS_SESSION_TOKEN`](#authentication): Supply a valid [`AWS_SESSION_TOKEN`](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_use-resources.html) ...
- [`S3_STORAGE_CLASS`](#amazon-s3-storage-classes): Specify the [Amazon S3 storage class](https://docs.aws.amazon.com/AmazonS3/latest/userguide/storage-class-intro.html)
```